Remotion/Core/Collections.DataStore/LazyLockingDataStoreAdapter.cs

```diff
@@ -98,26 +100,25 @@ public void Clear ()
       }
     }
 
+    [return: MaybeNull]
     public TValue GetValueOrDefault (TKey key)
     {
       ArgumentUtility.DebugCheckNotNull ("key", key);
 
       var result = _innerDataStore.GetValueOrDefault (key);
-      return result != null ? result.Value.Value : null;
+      return result != null ? result.Value.Value : null!;
     }
 
-    public bool TryGetValue (TKey key, out TValue value)
+    public bool TryGetValue (TKey key, [AllowNull, MaybeNullWhen (true)] out TValue value)

```

Comment:
Shouldn't this have `MaybeNullWhen (false)`?

```suggestion
    public bool TryGetValue (TKey key, [AllowNull, MaybeNullWhen (false)] out TValue value)
```